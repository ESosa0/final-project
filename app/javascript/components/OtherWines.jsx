import React from 'react'

const OtherWines = (props) => (
  <div className="other-wines">
    <p>Don't like <span className="titleize">{props.wineName}?</span></p>
    <p><span className="link" onClick={props.getNextWine}> See another wine</span> that goes nicely with your dish.</p>
  </div>
)

export default OtherWines