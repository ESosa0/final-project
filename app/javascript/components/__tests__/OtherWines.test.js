import OtherWines from '../OtherWines'
import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'

describe("OtherWines Component", () => {
  it("renders a default OtherWines component", () => {
    const component = shallow(<OtherWines/>)
    expect(toJSON(component)).toMatchSnapshot()
  })
})
