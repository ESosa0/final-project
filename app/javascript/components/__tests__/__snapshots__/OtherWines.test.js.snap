// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OtherWines Component renders a default OtherWines component 1`] = `
<div
  className="other-wines"
>
  <p>
    Don't like 
    <span
      className="titleize"
    >
      ?
    </span>
  </p>
  <p>
    <span
      className="link"
    >
       See another wine
    </span>
     that goes nicely with your dish.
  </p>
</div>
`;
